packages/e2e/address.js

```diff
@@ -1,3 +1,9 @@
-const { address1055 } = require("@vergunningcheck/mocking/src/address");
-
+// const { address1055 } = require("@vergunningcheck/mocking/src/address");
```

Comment:
Can you replace this comment? Maybe explain why we don't import from the mocking package here. Therefore some duplicate code here is acceptable.